src/main/java/org/folio/service/organization/OrganizationStorageService.java

```diff
@@ -9,7 +9,6 @@
 import static org.folio.util.ResourcePathResolver.resourcesPath;
 
 import java.util.Collections;
-import java.util.HashSet;
 import java.util.Map;
```

Comment:
looks it is not used somewhere

Reply:
If you see some unused imports - feel free to delete them